pkg/util/policy/agent/agent.py

```diff
@@ -56,6 +47,9 @@ def filter_rules_idx(rules):
 # We want to receive json object as a POST.
 class AgentHandler(BaseHTTPRequestHandler):
     def do_GET(self):
+        """
+        We do not have a storage so GET is not really supported.
```

Comment:
Ok, good point. It would have been a stop-gap measure anyway. I think in the end we want to have some sort of list of policies, which the agent can easily access and maintain, so that we can tell what's really implemented, locally. But this can wait.